data.sql

```diff
@@ -46,3 +46,42 @@ SET owner_id = CASE
     WHEN name IN ('Charmander', 'Squirtle', 'Blossom') THEN (SELECT id FROM owners WHERE name = 'Melody Pond')
     WHEN name IN ('Angemon', 'Boarmon') THEN (SELECT id FROM owners WHERE name = 'Dean Winchester')
   END;
+
+INSERT INTO vets(name, age, date_of_graduation) 
+    VALUES 
+        ('William Tatcher', 45, '2000-04-23'),
+        ('Maisy Smith', 26, '2019-01-17'),
+        ('Stephanie Mendez', 64, '1981-05-04'),
+        ('Jack Harkness', 38, '2008-06-08');
+
+INSERT INTO specializations (vet_id, species_id)
+VALUES
+   (1, 1),        
+   (3, 1),        
+   (3, 2),
+   (4, 2);
+
+NSERT INTO visits (animal_id, vet_id, date_of_visit)
```

Comment:
- [ ] So you have `date_of_visit` here but the schema file does not create that column. Please fix